manual/cypher/cypher-docs/src/docs/dev/ql/functions/index.asciidoc

```diff
@@ -283,3 +283,24 @@ include::split.asciidoc[leveloffset=+1]
 include::reverse.asciidoc[leveloffset=+1]
 
 include::tostring.asciidoc[leveloffset=+1]
+
+ifndef::asciidoctor[:leveloffset: 1]
+
+[[query-functions-udf]]
+== User-defined functions ==
+User-defined functions are written in Java, deployed into the database and are called in the same way as any other Cypher function.
+
+This example shows how you create a _user-defined function_ and invoke it from Cypher.
+[snippet,java]
+----
+component=neo4j-cypher-docs
+source=org/neo4j/procedure/example/JoinFunction.java
+classifier=sources
+tag=joinFunction
+----
+
+ifndef::asciidoctor[:leveloffset: 2]
+
+include::call-a-user-defined-function.asciidoc[]
```

Comment:
This needs to be `include::call-a-user-defined-function.asciidoc[leveloffset=+1]`